### ISSUE TITLE
Auto remove orphan dependencies

### ISSUE THREAD

**Issue by budhajeewa** [babun/babun#155]
I think we need an option for `pact` that allows us to automatically remove packages that were installed as dependencies of some package, but are no longer needed because that original package was uninstalled.

Basically, we need `pact` to do what `apt-get autoremove` does.

**Comment by tombujok** [comment]
Yes, it would be a great feature and not the difficult to implement.
Who is willing to contribute a patch :) ?
